app/models/vote.rb

```diff
@@ -0,0 +1,7 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+
+  validates_presence_of :user, :post
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.